docs/configuration.md

```diff
@@ -47,6 +47,7 @@ Property
 **envoy-control.envoy.snapshot.egress.handle-internal-redirect**                                            | Handle redirects by Envoy                                                                                                                                                                                   | false
 **envoy-control.envoy.snapshot.egress.host-header-rewriting.enabled**                                       | Enable rewriting Host header with value from specified header                                                                                                                                               | false
 **envoy-control.envoy.snapshot.egress.host-header-rewriting.custom-host-header**                            | Header name which value will override Host header                                                                                                                                                           | "x-envoy-original-host"
+**envoy-control.envoy.snapshot.egress.headers-to-remove**                                                   | List of headers to sanitize                                                                                                                                                                                 | empty list
```

Comment:
I'd duplicate the word `egress`/`ingress` in the description because right now it looks a bit like something was wrong there (a duplicate string)